### PR TITLE
Add ad validation metadata endpoint

### PR DESCRIPTION
**Pull request by charleystran**
## Description of change
Add an endpoint to provide the metadata for the address verification endpoints.

## Testing done
local testing against portal

## Acceptance Criteria (Definition of Done)
- [ ] . Metadata endpoint exists for the developer portal to call and allow version changes.

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

resolves https://github.com/department-of-veterans-affairs/vets-contrib/issues/4146

### CODE REVIEW

File: modules/veteran_verification/spec/requests/metadata_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Metadata Endpoint', type: :request do
+  describe '#get /metadata' do
+    it 'returns metadata JSON for address validation' do
+      get '/services/veteran_verification/address_validation_metadata'
+      expect(response).to have_http_status(:ok)
+      JSON.parse(response.body)
```
Comment (erluti) [inline modules/veteran_verification/spec/requests/metadata_request_spec.rb]: Personally I think specs are clearer when they have expectations, and if you expect it to not raise an exception there's an expectation to do that!
```
expect { JSON.parse(response.body) }.not_to raise_error
```

**Comment by edmangimelli** [comment]
lgtm, barring Philip's comment
💯

File: modules/veteran_verification/spec/requests/metadata_request_spec.rb (hunk 1 above)
Comment (charleystran) [reply modules/veteran_verification/spec/requests/metadata_request_spec.rb]: fixed this

**Comment by charleystran** [comment]
closing this PR because at this point we actually dont want to show v2 on the port yet

**Pull request closed without merging**